align/primitive/main.py

```diff
@@ -155,13 +164,20 @@ def generate_primitive(block_name, primitive, height=28, x_cells=1, y_cells=1, p
     assert isinstance(primitive, SubCircuit) \
         or primitive == 'generic' \
         or 'ring' in primitive, f"{block_name} definition: {primitive}"
+
     if primitive == 'generic':
         uc, _ = generate_generic(pdkdir, parameters, netlistdir=netlistdir)
     elif 'ring' in primitive:
         uc, _ = generate_Ring(pdkdir, block_name, x_cells, y_cells)
     elif 'MOS' == primitive.generator['name']:
+        style = None
```

Comment:
@kkunal1408 I look at this dictionary for the style parameter, but you can do this differently if that makes sense.

Reply:
Instead of string manipulation by doing "name"+"style" I prefer adding a separate name. If this does not work for you, you can uncomment the commented lines.

Reply:
group_block constraint extension was modified such that it helps another PR 1079